docs/.vuepress/config.js

```diff
@@ -112,15 +112,18 @@ function getGuideSidebar(guide, api, bestPractices, mobile, resources) {
       title: mobile,
       collapsable: false,
       children: [
+        'mobile-getting-started',
+        'mobile-best-practices',
         'dapp-compatibility',
-
+        'deeplinking',
```

Comment:
Suggestion for structure:

1. Getting started
2. Make your site compatible with MetaMask  
3. Best practices 
4. Enhance your users' experience with your site/app 
(deeplinking, wallet connect)